shared/clinical-code-sets/medications/female-sex-hormones/1/female-sex-hormones.snomed.txt

```diff
@@ -674,3 +674,57 @@
 36394711000001107	Progesterone 25mg/1.112ml solution for injection vials
 36394811000001104	Lubion 25mg/1.112ml solution for injection vials (Pharmasure Ltd)
 36395011000001109	Lubion 25mg/1.112ml solution for injection vials (Pharmasure Ltd)
+11391011000001107	Testosterone 300micrograms/24hours patches (product)
```

Comment:
I don't think testosteron should be in this code set. If that's right then remove the ~15 instances. Also checkout the things like **Testo**derm, **Test**ogel, **Test**im as they sound a bit like testosterone products.